tsfile/src/main/java/org/apache/iotdb/tsfile/utils/BytesUtils.java

```diff
@@ -523,7 +523,7 @@ public static void longToBytes(long srcNum, byte[] result, int pos, int width) {
         width -= m;
         int mask = 1 << (8 - cnt);
         cnt += m;
-        byte y = (byte) (srcNum >> width);
+        byte y = (byte) (srcNum >>> width);
```

Comment:
Add a UT for this bug?

Besides, `intToBytes` seems have a same bug...

Reply:
The UT tests for this bug have been added.